cli/cmd/install.go

```diff
@@ -470,5 +470,15 @@ func errorAndCleanup(err error, kubeClient kubernetes.Interface) error {
 	return err
 }
 
+func needsDeletion(existingKey, sgxKey string) bool {
```

Comment:
I don't understand this function.
Doesn't that return true for all non-sgx related values?
Can you make a comment for the second check? -> Reads like we want to delete the `sgx.intel.com/epc` value if the intel plugin is detected?